server/gfmd/user.c

```diff
@@ -36,6 +36,11 @@
 #define USER_DN_HASHTAB_SIZE	3079	/* prime number */
 
 /* in-core gfarm_user_info */
+struct user_auth {
```

Comment:
We would like to consider the implementation here later, so please remove the addition of this struct user_auth definition.